KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/LanguageDetector.java

```diff
@@ -0,0 +1,55 @@
+package org.sleuthkit.autopsy.keywordsearch;
+
+import com.optimaize.langdetect.LanguageDetectorBuilder;
+import com.optimaize.langdetect.i18n.LdLocale;
+import com.optimaize.langdetect.ngram.NgramExtractors;
+import com.optimaize.langdetect.profiles.LanguageProfile;
+import com.optimaize.langdetect.profiles.LanguageProfileReader;
+import com.optimaize.langdetect.text.CommonTextObjectFactories;
+import com.optimaize.langdetect.text.TextObject;
+import com.optimaize.langdetect.text.TextObjectFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+class LanguageDetector {
+
+  public enum Language {
```

Comment:
I find the existence of 2 different enums named "Language" to be confusing. Is it possible to consolidate these enums and possible use Enum.name() and toString() methods in place of static strings (e.g. "en" and "ja" below)?

Reply:
I consolidated the two `Language` classes. Now there is only `JAPANESE`.  A `Language` is detected means the document should be indexed to a language-specific field such as `content_ja`.